admin-policy.md

```diff
@@ -0,0 +1,83 @@
+3. All discussion should happen on this PR, and on no other mediums. The entire community should participate in this discussion, not just the admins. After discussion has died down, the admins can choose to merge/reject a PR. In either case, if there was any opposition or unaddressed points in the comments, the admins *must* address all points and sides of the argument when merging.
+4. Note that formatting, grammar, or simple clarification PRs do not need to go through this process. They can be made as normal PRs against the repo, and an admin can merge them at any time.
+5. Any changes to a PR before merging should ideally be made by the author of the PR. Pull requests by others to PRs are generally not allowed. The exception to this is when permission is given by the PR author due to the author being unable to make the discussed changes at that time.
+6. Note that commits should never be removed from PRs (i.e. through rebasing/squashing), as we would like to preserve the entire history of the PR.
+
+## 3. Management of Administration
+
+1. We will increase the number of admins per term as the current admin team sees fit. A general indicator of how many admins are needed is to see how long it takes posts to be moderated. If all bad posts are caught within some small amount of time (i.e. 30 minutes) of posting, then we have enough admins.
+2. The length of a term for an admin will be 3 months.
+3. An admin can serve for a maximum of two consecutive terms, but there is no maximum limit on the number of terms an admin can serve for.
+4. All admin activity will be public. In the case of bans, see the ban policy later in this document. For larger, more official meetings, either the entire chat log will be posted (possibly as a GitHub commit or a gist), or paraphrased meeting minutes. Small, one-off conversations/meetings do not need to be public. Ideally, these meetings should occur in the `##hx-admin` IRC channel on Freenode.
+5. It is *highly* recommended that admins idle on `##hx-admin` so that the community can easily have a short discussion or ask questions directly to the admins in a public area, without polluting the main HX group. HX Meta is also alright for this purpose, but an IRC conversation can often provide immediate feedback.
+6. Admins are not allowed to deactivate their Facebook accounts during their term. Doing so is cause for removal as an admin.
+
+## 4. Election of Admins
+
+1. The current admin team will determine how many admins are elected for the next term. 
+2. 14 days before the end of a term, the web interface will allow candidates to mark themselves as running. 
+  - Candidates must be nominated by three group members via the web interface (after they have marked themselves as running) to be put on the ballot. Each group member can nominate one member per term.
```

Comment:
"- Candidates must be nominated by three group members via the web interface (after they have marked themselves as running) to be put on the ballot. Each group member can nominate one member per term.
 +3. 7 days before the end of a term, the web interface will allow group members to vote for a candidate. "

A candidate meaning 1 candidate?
This currently reads as 3 nominations to run, everyone gets 1 nomination and 1 vote.